### PR TITLE
Renaming of Groups

### PR DESCRIPTION
**Pull request by InfiniteCoder06**
This is one of the most requested feature mentioned in #27

### CODE REVIEW

File: app/src/main/java/com/beemdevelopment/aegis/ui/GroupManagerActivity.java [hunk 1]
```diff
@@ -126,12 +169,19 @@ private void saveAndFinish() {
 
             saveAndBackupVault();
         }
+        if (!_renamedGroups.isEmpty()) {
+            for (VaultGroup renamedGroup : _renamedGroups) {
+                _vaultManager.getVault().renameGroup(renamedGroup);
```
Comment (michaelschattgen) [inline app/src/main/java/com/beemdevelopment/aegis/ui/GroupManagerActivity.java:174]: This causes a crash when you remove the group that you just renamed.

**Review by michaelschattgen — CHANGES_REQUESTED** [review]
Thanks for your contribution! Close, but a small fix is needed before we can merge this.

File: app/src/main/java/com/beemdevelopment/aegis/ui/GroupManagerActivity.java (hunk 1 above)
Comment (InfiniteCoder06) [reply app/src/main/java/com/beemdevelopment/aegis/ui/GroupManagerActivity.java:174]: Thanks for the info, Will be looking onto it.

**Comment by michaelschattgen** [comment]
Perfect, thank you! Could you please squash your commits so we can merge this into master?

**Review by michaelschattgen — APPROVED** [review]

**Pull request merged**